lib/rails/generators/neo4j/install/templates/neo4j_rails.rb

```diff
@@ -0,0 +1,8 @@
+# Enable Neo4j generators, e.g:  rails generate model Admin --parent User
+config.generators do |g|
+  g.orm             :neo4j
+  g.test_framework  :rspec, :fixture => false
```

Comment:
We use `rubocop` in the project, so I would run:

    rubocop -aD

and fix anything that it doesn't automatically fix before committing the changes.

I think that Travis is still failing for other reasons (maybe because of the things that @subvertallchris was still going to fix?)